inst/tutorials/introduction/introduction.Rmd

```diff
@@ -0,0 +1,265 @@
+
+
+![](images/malsim_overview.png){width=100%}
+
+We will look at each of these in turn.
+
+
+## 1: Generate parameter set
+
+The key function here is `get_parameters()`
+
+Try running it to see what it generates.
+
+```{r get_parameters, exercise=TRUE}
+get_parameters()
+```
+
+You should see a long list of parameters - it's a complicated model!
+
+To understand what each parameter does in the model, you can go to the [help page](https://mrc-ide.github.io/malariasimulation/reference/get_parameters.html) (or use `?get_parameters()` when you are in R to see the same information).
+
+
+Please don't try to understand all of these right now! Instead, use the help page to try answering the questions below for now, and if there are any parameters you'd like to know more about, feel free to ask a demonstrator.
+
+
+```{r quiz}
+quiz(caption = "Parameters Qs",
+     question_numeric(
+       "How many days does an untreated asymptomatic infection last for (A -> U)?",
+       answer(195, correct = TRUE), allow_retry = T, correct = ("That's right! On average, asymptomatic infections last for 195 days before they become sub-patent infections.")),
+     question("What is the model's default mosquito species?",
+              answer("An. arabiensis"),
+              answer("An. funestus"),
+              answer("An. gambiae", correct = TRUE),
+              answer("An. stephensi"), allow_retry = T),
+     question_numeric("What is the default adult mosquito mortality rate (to 3 dp)?",
+                      answer("0.132", correct = TRUE), allow_retry = T, correct = "That's right! The parameter is called `mum`."),
+     question_numeric("What is the default human population size?",
+                      answer("100", correct = TRUE), allow_retry = T, correct = "That's right! 100 is really quite a small population, but it's worth bearing in mind that simulations with larger populations take a longer time to run."),
+     question_text("What is the default mosquito population size?",
+                   answer("1000", correct = TRUE), allow_retry = T),
+     question_numeric("What is the default initial susceptible human state proportion (to 2 dp)?",
+                      answer("0.42", correct = TRUE), allow_retry = T),
+     question("How much immunity to malaria does the initial population have?",
+              answer("0", correct = TRUE),
+              answer("2.4"),
+              answer("20"),
+              answer("53"), allow_retry = T,
+              correct = "That's right! The population begins with no immunity. This isn't realistic for most settings, and we will describe how to adjust this during the tutorial."
+     )
+)
+
+```
+
+
+Any of these parameters can be adjusted within the `get_parameters()` function using the overrides argument. For example, we can increase the human population size (`human_population`) like this:
+
+```{r parameters_override, exercise=TRUE}
+get_parameters(overrides = list(human_population = 1000))
```

Comment:
This might keep the session a bit neater?
```suggestion
parameters <- get_parameters(overrides = list(human_population = 1000))
parameters$human_population
```